wiki/Game_modifier/id.md

```diff
@@ -13,21 +13,21 @@ tags:
 
 # Game modifier
 
-*Untuk penggunaan lainnya, lihat [Mod (disambiguasi)](/wiki/Disambiguation/Mod).*
+*Untuk penggunaan istilah "mod" dalam konteks-konteks lainnya, lihat [Mod (disambiguasi)](/wiki/Disambiguation/Mod).*
 
 ![Layar pemilihan mod](img/mod-selection-screen-ID.jpg "perbandingan layar pemilihan mod antara mode permainan osu! (kiri atas), osu!taiko (kanan atas), osu!catch (kiri bawah), dan osu!mania (kanan bawah)")
 
-**Game modifier** (yang pada umumnya disingkat sebagai "mod") merupakan elemen permainan yang dapat mengubah ataupun memodifikasi tampilan serta tingkat kesulitan dari suatu [beatmap](/wiki/Beatmap). Mod-mod yang ada di osu! dapat diaktifkan dari `Layar Pemilihan Mod` (sebagaimana yang tertera di atas), di mana penggunaan suatu mod dapat membuat beatmap menjadi lebih mudah, lebih sulit, atau murni ditujukan untuk kesenangan semata.
+**Game modifier** (atau yang pada umumnya disingkat sebagai "mod") merupakan elemen permainan yang dapat mengubah ataupun memodifikasi tampilan serta tingkat kesulitan dari suatu [beatmap](/wiki/Beatmap). Mod-mod yang ada di osu! dapat diaktifkan dari `Layar Pemilihan Mod` (sebagaimana yang tertera di atas), di mana penggunaan suatu mod dapat membuat beatmap menjadi lebih mudah, lebih sulit, atau murni ditujukan untuk kesenangan semata.
 
-Pemain dapat mengakses `Layar Pemilihan Mod` pada layar pemilihan lagu dengan mengklik tombol `Mods` di pojok kiri bawah layar atau dengan menekan `F1`. Setelah memasuki `Layar Pemilihan Mod`, Shortcut untuk memilih mod tertentu juga tersedia. Shortcut tersebut dapat diubah melalui pengaturan.
+Para pemain dapat mengakses `Layar Pemilihan Mod` pada layar pemilihan lagu dengan mengklik tombol `Mods` yang terletak pada pojok kiri bawah atau dengan menekan `F1`. Suatu mod dapat diaktifkan secara cepat dengan menekan tombol *shortcut* yang sesuai dengan yang ada pada `Layar Pemilihan Mod`, di mana tombol *shortcut* ini dapat diatur pada menu Options apabila dikehendaki.
 
-Mod dibagi menjadi tiga kategori: `Pengurang Kesulitan`, `Peningkat Kesulitan`, dan `Spesial`, dan mereka juga dapat mengurangi, menambah, atau menghapus [`Pengali Skor`](/wiki/Game_modifier/Score_multiplier). Ketika dua mod digunakan secara bersamaan, pengali skor tersebut akan dikalikan bersama (misalnya, `1.06x * 1.12x = 1.1872x`).
+Mod-mod yang ada di osu! dibagi menjadi tiga kategori: `Pengurang Kesulitan`, `Peningkat Kesulitan`, dan `Spesial` yang dapat mengurangi, menambah, atau bahkan menonaktifkan [`Pengali Skor`](/wiki/Game_modifier/Score_multiplier) sepenuhnya. Pada saat dua atau lebih mod diaktifkan secara bersamaan, efek-efek pengali skor yang aktif juga akan ikut dikalikan bersama (misalnya, `1.06x * 1.12x = 1.1872x`).
```

Comment:
```suggestion
Mod-mod yang ada di dalam permainan osu! dibagi menjadi tiga kategori: `Pengurang Kesulitan`, `Peningkat Kesulitan`, dan `Spesial` yang dapat mengurangi, menambah, atau bahkan menonaktifkan [`Pengali Skor`](/wiki/Game_modifier/Score_multiplier) sepenuhnya. Pada saat dua atau lebih mod diaktifkan secara bersamaan, efek-efek pengali skor yang aktif juga akan ikut dikalikan bersama (misalnya, `1.06x * 1.12x = 1.1872x`).
```

Add "di dalam permainan" to add more clarity that applied to all game mode.

Reply:
👍